neqo-transport/src/connection/idle.rs

```diff
@@ -46,20 +46,14 @@ impl IdleTimeout {
         self.timeout = min(self.timeout, peer_timeout);
     }
 
-    pub fn expiry(&self, now: Instant, pto: Duration, keep_alive: bool) -> Instant {
+    pub fn expiry(&self, now: Instant, pto: Duration) -> Instant {
         let start = match self.state {
             IdleTimeoutState::Init => now,
             IdleTimeoutState::PacketReceived(t) | IdleTimeoutState::AckElicitingPacketSent(t) => t,
         };
```

Comment:
It might pay to factor this into a function now to save repetition.

Reply:
Will do.